indra/tools/reading/read_pmids.py

```diff
@@ -445,11 +445,19 @@ def get_stmts(pmids_unread, cleanup=True, sparser_version=None):
     return stmts
 
 
+def get_stmts_from_cache(pmid):
+    json_str = s3_client.get_reader_json_str('sparser', pmid)
+    stmts = []
+    if json_str is not None:
+        stmts = sparser.process_json_dict(json.loads(json_str))
```

Comment:
I haven't tested this but this doesn't look right - wouldn't sparser.process_json_dict return a SparserProcessor, not a list of Statements? Has this worked in practice?

Reply:
Ah, yes, it does. I misread it when I saw `sp.get_statements()`.

Reply:
Ok...this got much harder to push the fix because of the rebase...but I have the fix ready.